Octokit.Tests.Integration/Clients/IssuesClientTests.cs

```diff
@@ -233,6 +242,22 @@ public IssuesClientTests()
     }
 
     [IntegrationTest]
+    public async Task CanRetrieveIssueWithMultipleAssignees()
+    {
+        var issue = await _issuesClient.Get("octokit", "octokit.net", 1171);
+
+        Assert.True(issue.Assignees.Count == 2);
+    }
+
+    [IntegrationTest]
+    public async Task CanRetrieveIssuesWithMultipleAssignees()
+    {
+        var issues = await _issuesClient.GetAllForRepository("octokit", "octokit.net");
+
+        Assert.True(issues.Any(x => x.Assignees.Count > 0));
```

Comment:
Given `1171` is now a closed issue, is it worth keeping this around? Could we build a fresh repository to test `GetAll`, rather than querying for all the issues in the Octokit repository?

If not, I like this sort of test:

```csharp
var request = new RepositoryIssueRequest { State = ItemStateFilter.All };

var issues = await _issuesClient.GetAllForRepository("octokit", "octokit.net", request);

Assert.True(issues.Any(x => x.Assignees.Count > 1));
```